CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Added: `at-rule-name-newline-after` rule.
 - Fixed: `function-url-quotes` ignores values containing `$sass` and `@less` variables.
 - Fixed: selector-targeting rules ignore Less mixins and extends.
+- Fixed: no-extra-semicolons reports errors on the correct line
```

Comment:
@epmatsw Thanks for fixing this bug!

Sorry to be pedantic, but can we backtick the rule name and add a full stop please? Our release notes are generated from the changelog and so I don't want to forget to correct this before we do a release.